src/components/atoms/button/button.html.json

```diff
@@ -0,0 +1,3 @@
+{
+  "text": "Submit form"
```

Comment:
shouldn't buttons also have a "value"?

Reply:
They can, but they don't have to because they aren't inputs. A type does help though so I added one. This is kind of a bit of a stop gap really—It's going to take a while to get all form elements twiggfied properly.

Reply:
To be honest, I was including a button component mainly for the css so that we can extend it's styles from the link component.